crates/nu-command/src/charting/histogram.rs

```diff
@@ -186,13 +195,17 @@ fn run_histogram(
                             }
                         }
                     }
+                    // Propagate existing errors.
+                    Value::Error { error } => return Err(error),
                     _ => continue,
                 }
             }
 
             if inputs.is_empty() {
                 return Err(ShellError::UnsupportedInput(
-                    format!("expect input is table, and inputs doesn't contain any value which has {col_name} column"),
+                    format!("none of the input values had a '{col_name}' column"),
+                    "".into(),
+                    head_span, // MUSTFIX
```

Comment:
Were you planning on fixing this one up before submitting the PR?

Before:
![image](https://user-images.githubusercontent.com/26268125/209208025-a67b0cbd-4c06-4779-ae2e-859be535a158.png)

After:
![image](https://user-images.githubusercontent.com/26268125/209207881-00de89de-827d-4e02-8bb9-a12f72fc45a8.png)